ndmg/scripts/ndmg_bids.py

```diff
@@ -447,16 +453,18 @@ def main():
     # it's super gross.
     if buck is not None and remo is not None:
         if subj is not None:
-            if len(sesh) == 1:
-                sesh = sesh[0]
+            #if len(sesh) == 1:
+            #    sesh = sesh[0]
```

Comment:
Did you test this? I remember this conditional being added because if there was only one session, it wouldn't be a list, and later parts of the pipeline would index it wrong.